examples/bql/example_3.bql

```diff
@@ -44,5 +56,52 @@ WHERE {
 }
 HAVING NOT(?offspring = ?grandchildren);
 
-# Drop the graph.
-DROP GRAPH ?family;
+# Which parents have name coming lexicographically after "joe"?
+SELECT ?parent_name
+FROM ?family
+WHERE {
+  ?parent ID ?parent_name "parent_of"@[] ?child
+}
+GROUP BY ?parent_name
+HAVING ?parent_name > "joe"^^type:text;
+
+# Who has height smaller than 173 cm?
+SELECT ?person, ?height
+FROM ?info
+WHERE {
+  ?person "height_cm"@[] ?height
+}
+HAVING ?height < "173"^^type:int64;
+
+# Who has ever bought a car "model y" or "model z"?
+SELECT ?person
+FROM ?info
+WHERE {
+  ?person "bought"@[,] ?product
+}
+HAVING (?product = /car<model y>) OR (?product = /car<model z>);
+
+# What are the triples in the ?info graph having either "height_cm"@[]
+# or "born_in"@[] as predicate?
+SELECT ?subj, ?pred, ?obj
+FROM ?info
+WHERE {
+  ?subj ?pred ?obj
+}
+HAVING (?pred = "height_cm"@[]) OR (?pred = "born_in"@[]);
+
+# N.B. for nodes and predicates you are allowed to use equalities inside the
```

Comment:
N.B stands for what?

Reply:
["Nota bene"](https://pt.wikipedia.org/wiki/Nota_bene#:~:text=Nota%20bene%20%E2%80%93%20Wikip%C3%A9dia%2C%20a%20enciclop%C3%A9dia%20livre), in the sense of "pay attention to". If you want me to change it for something else, just lmk.